Formula/gnu-apl.rb

```diff
@@ -20,20 +20,34 @@ class GnuApl < Formula
     depends_on "libtool" => :build
   end
 
+  option "with-postgresql", "Build with Postgresql support"
+
   # GNU Readline is required; libedit won't work.
   depends_on "readline"
   depends_on :macos => :mavericks
 
+  depends_on :postgresql => :optional
+
   def install
     # Work around "error: no member named 'signbit' in the global namespace"
     # encountered when trying to detect boost regex in configure
     ENV.delete("SDKROOT") if DevelopmentTools.clang_build_version >= 900
 
+    args = %W[
+      --disable-debug
+      --disable-dependency-tracking
+      --disable-silent-rules
+      --prefix=#{prefix}
+    ]
+
+    if build.with? "postgresql"
+      ENV.prepend "LDFLAGS", "-L#{Formula["postgresql"].opt_lib}"
```

Comment:
instead `args << "--with-postgresql=#{Formula["libpq"].opt_bin}/pg_config"`

Reply:
and actually it defaults to on so you don't even need to pass this argument to configure or set any of the flags manually